src/app/loadout/loadout-ui/LoadoutMods.tsx

```diff
@@ -72,6 +72,15 @@ export default memo(function LoadoutMods({
             })}
             key={getModRenderKey(mod)}
             plug={mod}
+            onClose={
+              onUpdateMods &&
+              (() => {
+                const firstIndex = savedMods.findIndex((savedMod) => savedMod.hash === mod.hash);
+                const newMods = [...savedMods];
+                newMods.splice(firstIndex, 1);
+                onUpdateMods(newMods);
+              })
+            }
```

Comment:
Do you think it's worth pulling out the single mod being rendered here so we can `useCallback` this? I was a bit undecided.

Reply:
Probably?

Reply:
Done